README.md

```diff
@@ -130,6 +130,41 @@ bashcompinit
 source <path/to/ddev-completion.sh>
 ```
 
+#### Functional Test Code Coverage
+
+The following dependencies are needed to run functional code coverage:
+
+Chromedriver 2.14:
+```
+https://chromedriver.storage.googleapis.com/index.html?path=2.14/
```

Comment:
Is 2.14 a strict requirement? Or would the latest version work (2.20)? Just wondering because there's a brew formula to install this that would be easier than doing it manually:

```bash
brew install chromedriver
```